apps/scan/frontend/src/screens/poll_worker_screen.tsx

```diff
@@ -348,7 +348,7 @@ function PollWorkerScreenContents({
       isAfterPollsTransition: true,
       printResult,
     });
-    onPollsClose();
+    voterSettingsControls.resetVoterSettings();
```

Comment:
This approach won't actually work right now, because `voterSettingsControls.resetVoterSettings()` will not reset the voter session state that is managed by the invocation of `useVoterSettingsControl` from `app_root`